File: relationship-manager/admin-service/src/main/java/com/bank/rm/admin/AdminServiceApplication.java
package com.bank.rm.admin;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.admin", "com.bank.rm.common"})
public class AdminServiceApplication {
    public static void main(String[] args) { SpringApplication.run(AdminServiceApplication.class, args); }
}

File: relationship-manager/admin-service/src/main/kotlin/com/bank/rm/admin/AdminServiceApplication.kt
package com.bank.rm.admin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.admin", "com.bank.rm.common"])
class AdminServiceApplication

fun main(args: Array<String>) {
    runApplication<AdminServiceApplication>(*args)
}
